hack/jenkins/e2e.sh

```diff
@@ -40,7 +40,13 @@ function join_regex_no_empty() {
 }
 
 # GCP Project to fetch Trusty images.
-TRUSTY_IMAGE_PROJECT="${TRUSTY_IMAGE_PROJECT:-}"
+function get_trusty_image_project() {
+  gsutil cat "gs://trusty-images/image-project.txt"
+  # Clean up gsutil artifacts otherwise the later test stage will complain.
+  rm -rf .config &> /dev/null
+  rm -rf .gsutil &> /dev/null
+}
+TRUSTY_IMAGE_PROJECT="$(get_trusty_image_project)"
```

Comment:
I was wondering if this needs to be `export TRUSTY_IMAGE_PROJECT=...`, but I don't think it does. That variable isn't used in any other files, is it?